docs/dashboards/README.md

```diff
@@ -83,6 +83,162 @@ Note that the OTel collector can automatically merge configuration files for you
+            leg5[ ]:::hide-. inject k8s attributes .-> leg6[ ]:::hide
+            classDef hide height:0px
+        end
+    end
+
+    subgraph alignAlternativesSg[" "]
+        subgraph alt1[Self-monitoring collector]
+            attributesProcessorApi[k8s API]  
+
+            app1[Application w/ OTel]:::OpenTelemetryApp
+            app2[Application w/ OTel]:::OpenTelemetryApp
+            app3[Application w/ OTel]:::OpenTelemetryApp
+
+            otelcol1[OpenTelemetry Collector]:::OTelCollector
+            otelcol2[OpenTelemetry Collector]:::OTelCollector
+
+            selfmonGateway[Self-monitoring Collector with k8sattributesprocessor]:::SelfmonCollector
+
+            app1-->otelcol1
+            app2-->otelcol2
+            app3-->otelcol2
+
+            otelcol1-->otlpApi
+
+            otelcol1-.->selfmonGateway
+            otelcol2-.->selfmonGateway
+
+            otelcol2-->otlpApi
+
+            attributesProcessorApi -.  k8sattributesprocessor retrieves k8s attributes from the k8s API .- selfmonGateway
+
+            selfmonGateway-.->otlpApi
+        end
+
+        subgraph alt2[Direct k8s enrichment]
+            k8sInjection[k8s API]  
+            
+            app4[Application w/ OTel]:::OpenTelemetryApp
+            app5[Application w/ OTel]:::OpenTelemetryApp
+            app6[Application w/ OTel]:::OpenTelemetryApp
+
+            otelcol4[OpenTelemetry Collector]:::OTelCollector
+            otelcol5[OpenTelemetry Collector]:::OTelCollector
+
+            k8sInjection -.- otelcol4
+            k8sInjection -. inject k8s attributes as environment variables via k8s downwards API .- otelcol5
+
+            app4-->otelcol4
+            app5-->otelcol5
+            app6-->otelcol5
+
+            otelcol4-.->otlpApi
+            otelcol5-.->otlpApi
+            otelcol4-->otlpApi
+            otelcol5-->otlpApi
+        end
+    end
+
+classDef OpenTelemetryApp fill:#adc9ff,stroke:#1966FF
+classDef OTelCollector fill:#9afee0,stroke:#02D394
+classDef SelfmonCollector fill:#C2C2C2,stroke:#707070
+classDef AlternativeClass rx:100,fill:
+classDef hideSg fill:white,stroke:white
+classDef API fill:#ae70ff,stroke:#7F1AFF
+class alignAlternativesSg,alignOthersSg,dtOtlpApi hideSg
+class alt1,alt2 AlternativeClass
+
+linkStyle default stroke-width:3px
+linkStyle 1,7,8,11,17,18 stroke:#707070
+linkStyle 2,10,12,13 stroke:#C2C2C2,stroke-width:2px
+```
+
+### Gateway collector
+
+In this approach, collectors are configured to send their internal telemetry to one single dedicated collector, which enriches the incoming telemetry with k8s attributes using the `k8sattributesprocessor`.
+The `k8sattributesprocessor` retrieves this data from the k8s API, and attaches it to the telemetry passing through it.
+One limitataion of this approach is that the automatic enrichment will not work for the telemetry that that one collector instance (the "gateway" collector instance) exports itself.
```

Comment:
We have this docs page here: https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/collector/use-cases/kubernetes/k8s-enrich#kubernetes
I'll link to it, I think it could be helpful - thanks for pointing that out!